src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task{
```

Comment:
Since this is a class that extends tasks, it might be more suitable to name it TasksWithDeadline or something that shows that it's still a task instead of just a deadline